### PR TITLE
Added a separate section as a guide for embedding circuits in documentation/webpages

### LINKED ISSUE #262: Adding Documentation for embedding circuits for new contributors
## Is your feature request related to a problem? Please describe.
Currently there's no documentation as to how circuits should be embedded in the documentation for new contributors. I too felt the need of it when I started.

## Describe the solution you'd like
writing a separate section describing steps for the same

### PR DESCRIPTION
**Pull request by CapTen101**
Fixes #262 - Adding Documentation for embedding circuits for new contributors

#### Changes done:
- [x] Added a separate section - guide for embedding in documentation/webpages

#### Screenshots:
Screenshot from 2020-06-20 21-27-51
Screenshot from 2020-06-20 21-27-54
Screenshot from 2020-06-20 21-27-57
Screenshot from 2020-06-20 21-28-00

#### Preview Link(s):

https://deploy-preview-263--blissful-einstein-93d693.netlify.app/#/embedding_circuits

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by CapTen101** [comment]
Please review @YashKumarVerma

File: docs/embedding_circuits.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Embedding Circuits
+
+> This section is a helpful step-by-step guide for new contributors trying to improve the CircuitVerse Documentation by embedding circuits.
+
+Contributing Authors: [@CapTen101](https://github.com/CapTen101)
```
Comment (YashKumarVerma) [inline docs/embedding_circuits.md:5]: we're discontinuing this practice, all contributors would be listed in the project readme/credits page.

File: docs/embedding_circuits.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+These circuits can be embedded anywhere (apart from the documentation) since they utilize `iframe` HTML tag generated from the Circuitverse [website](circuitverse.org).
```
Comment (YashKumarVerma) [inline docs/embedding_circuits.md:7]: add http before link, else relative link would open

File: docs/embedding_circuits.md (hunk 1 above)
Comment (CapTen101) [reply docs/embedding_circuits.md:5]: Oh, okay. I wasn't aware of that. I remove that

File: docs/embedding_circuits.md (hunk 2 above)
Comment (CapTen101) [reply docs/embedding_circuits.md:7]: done

**Comment by CapTen101** [comment]
Kindly review. I've done the changes @YashKumarVerma

**Comment by CapTen101** [comment]
@Shivansh2407 Regarding the 2) point you mentioned:

Screenshot actually doesn't capture mouse pointer in linux. But if we do want to emphasize things in the tutorial images then we can maybe enclose them in a red box. That will highlight them greatly :+1: 
Let me know if this is right

**Comment by Shivansh2407** [comment]
Sure. Do that and squash the commits after fixing the issues @CapTen101 👍

**Comment by CapTen101** [comment]
Ok, right @Shivansh2407 👍

**Comment by CapTen101** [comment]
@Shivansh2407 I've insert a red marker in all the images. 
One thing is that I'm not able to squash those commits with "Merge branch master of ...". I cannot see them in the `git log` panel.

**Comment by Shivansh2407** [comment]
Check this out if you need help with squashing the commits : https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by CapTen101** [comment]
Yes I know how to squash commits :smile:  Have done it many times below. But this time, while squashing the other 8 commits never appear in the editor where we have to change the `pick` to `squash` keyword @Shivansh2407

**Comment by CapTen101** [comment]
@Shivansh2407 These commits are from previous merges by pulling the work from master branch

**Comment by CapTen101** [comment]
I think you have to merge this way only. The next time I pull the changes I would have to `git pull --rebase` to avoid these unnecessary merge commits. @Shivansh2407

**Comment by CapTen101** [comment]
I've messed this up pretty badly. I'll close this PR and open another one. @YashKumarVerma @Shivansh2407

**Comment by CapTen101** [comment]
Actually the problem is that my master is already 8 commits ahead of this repository's master branch :frowning_face:

**Pull request closed without merging**